recipe/meta.yaml

```diff
@@ -23,16 +20,34 @@ requirements:
     - jinja2
   run:
     - python >=3.5
+    - importlib-metadata
```

Comment:
No, I disagree with this. fsspec tries hard not to have any dependencies at all (otherwise I would have used `entrypoints`).
```python
    try:
        from importlib_metadata import entry_points
    except ImportError:
        entry_points = None
```
should not error if the package is missing. Of course, for python>=3.8, it's guaranteed to exist.